tools/scripts/ansible/cldr-apps-playbook.yml

```diff
@@ -47,6 +47,13 @@
         # this is deep because we will need to keep updating
         # it with history. It does not include LFS as that
         # is not needed for the surveytool.
+    - name: Change ownership of CLDR trunk to surveytool
+      file:
+        path: "{{ cldr_trunk_path }}"
+        owner: surveytool
+        group: root
+        state: directory
+        recurse: yes
```

Comment:
ansible-lint would like "mode: ..." added here -- but that's complicated with "recurse: yes"

Reply:
Actually we might skip the "Change ownership" task and instead add "become_user: surveytool" to the previous task -- I thought it wasn't working when I tried it before, but that may have been due to confusion between /etc/tomcat and /var/lib/tomcat

Reply:
Yes, that works. I'll make a new PR.